script/dynamicwindowutils.js

```diff
@@ -59,7 +60,7 @@ define(
       addEventCallback(undefined, detectIfUnpaused);
 
       function detectIfUnpaused (event) {
-        if (event.state !== MediaPlayerBase.STATE.PAUSED) {
+        if (event.state !== MediaPlayerBase.STATE.PAUSED && event !== MediaState.PAUSED) {
```

Comment:
This change is a bit of a code smell, as it shows that this is working with 2 different event models. Can't think of a nice way to solve this off the top of my head I'm afraid.